app/views/about_users/edit.html.haml

```diff
@@ -0,0 +1,3 @@
+= form_for [@user, @about_user] do |f|
```

Comment:
Should add `.container` as well somewhere in this file. Refer to other pages

Reply:
👍